ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -150,12 +150,36 @@ export const getUpdatedProfileValue = (
   }
 };
 
+export const isCompleted = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.completionEpochMillis;
+
 export const isBypassed = (
   profile: Profile,
   moduleName: AccessModule
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
+export const isExpired = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.expirationEpochMillis;
+
+export const displayModuleStatus = (
+  profile: Profile,
+  moduleName: AccessModule
+): string =>
+  isCompleted(profile, moduleName)
```

Comment:
this logic is not correct (example: expired modules are also complete: we need to show "Expired").  Please reuse the logic from access-renewal

Reply:
Done